setup.py

```diff
@@ -50,7 +50,7 @@ def run_tests(self):
 
 setup(
     name="dtale",
-    version="1.7.10",
```

Comment:
please add `redislite` to `tests_require` and I'd also update `extras_require` to this:
```
extras_require={
    'arctic': ["arctic"],
    'r': [
        "rpy2<=2.8.6; python_version < '3.0'",
        "rpy2; python_version > '3.0'",
    ],
    'redis': ['redislite'],
},
```
Also, not sure if you want to add an entry for shelve

Reply:
I think it should be okay without shelve since it's part of the standard library? (in both 2.7 and 3.6)